packages/reports/src/Stores/index.ts

```diff
@@ -1,10 +1,20 @@
 import ModulesStore from './Modules';
+import ProfitTableStore from './Modules/Profit/profit-store';
+import StatementStore from './Modules/Statement/statement-store';
+import type { TStores } from '@deriv/stores';
+
+export type TRootStore = TStores & {
+    modules: {
+        profit_table: ProfitTableStore;
+        statement: StatementStore;
+    };
+};
 
 export default class RootStore {
-    constructor(core_store) {
+    constructor(core_store: any) {
```

Comment:
> 

Do you mean I have to add some properties to TRootStore in packages/stores/types.ts ? I will need to add these:

gtm;
rudderstack;
pushwoosh;
notifications;
contract_replay;
contract_trade;
portfolio;
chart_barrier_store;
active_symbols;

Are you sure? Not inside of any store type, I have to create 9 new store types in packages/stores/types.ts?

Reply:
@sergei-deriv Well yes 😁😈 Unfortunately all of those stores exist in the `root_store` inside the `@deriv/core` package.
But this will be too much to be done by only one person, That’s why I keep suggesting first using the `TStore` inside packages to have safer types before starting to migrate components so then each person can start adding a small part of the stores and at the end, we will have everything typed.

---

@niloofar-deriv @shayan-deriv can you guys advise on this if this should be done by @sergei-deriv in this PR or if we can leave it for others? I mean he already added the base setup for using the `TStore` type which is nice if we can get it merged and then others start using it, But if we want to do everything in this PR then it will take a long time.

Reply:
@farzin-deriv Let's leave these changes for now and refactoring them later :smile: 
We need to refactor other components too, and these changes I think is out of the scope of this PR

Reply:
> @farzin-deriv Let's leave these changes for now and refactoring them later 😄 We need to refactor other components too, and these changes I think is out of the scope of this PR

Thank you, I fully agree to leave these changes for now and not create new store types :)